examples/rl_gsm8k/orchestrate_rl.py

```diff
@@ -318,6 +318,10 @@ def main(cfg: DictConfig):
     run = init_wandb(cfg, exp_path, flatten_dict_config(cfg))
     if run is None:
         raise ValueError("Failed to initialize wandb run")
+    
+    # Add migration call here
```

Comment:
Can you please remove this, see an explanation in my other comment

Reply:
same as above